Softeq.XToolkit.Common.iOS/Extensions/NSStringExtensions.cs

```diff
@@ -79,6 +79,23 @@ public static NSMutableAttributedString BuildAttributedString(this string input)
             return self;
         }
 
+        public static NSMutableAttributedString LinksFont(
```

Comment:
Where is it used? For `UITextView` we can use `WeakLinkTextAttributes` to do this.

Reply:
It didn't work, so I had to change Attributed text itself.

Reply:
https://stackoverflow.com/questions/35053694/uitextview-linktextattributes-font-attribute-not-applied-to-nsattributedstring